tests/sqlparser_postgres.rs

```diff
@@ -1467,3 +1467,30 @@ fn pg_and_generic() -> TestedDialects {
         dialects: vec![Box::new(PostgreSqlDialect {}), Box::new(GenericDialect {})],
     }
 }
+
+#[test]
+fn parse_escaped_literal_string() {
+    let sql = r#"SELECT E's1 \n s1', E's2 \\n s2', E's3 \\\n s3', E's4 \\\\n s4', E'\''"#;
```

Comment:
It's not a correct value because the last quote was escaped, there is no single quote which should close the string expr. https://github.com/sqlparser-rs/sqlparser-rs/pull/502/commits/f46b07e7ce383148c1fcb8755cd37965f0553708